### PR TITLE
machine_configuration: Update README.md

### PR DESCRIPTION
**Pull request by hershpa**
Optimize and clean up the readme content for clarity. Signed-off-by: Hersh Pathak hersh.pathak@intel.com

### CODE REVIEW

**Comment by hershpa** [comment]
Hi @mregmi and @uMartinXu, can you please help review this PR?

File: machine_configuration/README.md [hunk 1]
```diff
@@ -1,51 +1,75 @@
...
+```
 
-Following command creates a custom machine config pool for worker nodes with Intel QAT feature. These nodes are labeled `intel.feature.node.kubernetes.io/qat: 'true'` by NFD. 
+To verify the MachineConfigPool has been successfully created, navigate to the Compute -> MachineConfigPools section in the web console or run `oc get mcp` via CLI. Ensure `intel-qat` MachineConfigPool is present.
 
-```$ oc apply -f https://github.com/intel/intel-technology-enabling-for-openshift/blob/main/machine_configuration/intel-qat-machine-config-pool.yaml```
+* Turn on `intel_iommu` kernel parameter for QAT
 
-* Turn on intel_iommu kernel parameter for QAT
+For QAT, `intel_iommu` is a kernel parameter that needs to be turned on using the following command.
```
Comment (mregmi) [inline machine_configuration/README.md:67]: should be called "kernel boot parameter"

**Review by mregmi — CHANGES_REQUESTED** [review]
one comment. everything else looks good

File: machine_configuration/README.md (hunk 1 above)
Comment (hershpa) [reply machine_configuration/README.md]: Thanks, done.

**Pull request closed without merging**